routes/web.php

```diff
@@ -62,6 +63,10 @@
     Route::patch('/reservation/{reservation}/invoice/update', [InvoiceController::class, 'handleUpdateInvoice'])->name('invoice.update');
     Route::patch('/reservation/{reservation}/invoice/recalculate', [InvoiceController::class, 'handleRecalculateInvoice'])->name('invoice.recalculate');
 
+    // Cost adjustments
+
```

Comment:
Ik erger me aan deze witregel, bij de andere is dit er niet, niks belangerijk. Ben gewoon vreemd